ups/meas_extensions_simpleShape.table

```diff
@@ -5,10 +5,11 @@
 # included by low-level LSST packages such as utils or daf_base.
 # Any other package whose interface is used should be listed explicitly
 # rather than assuming it will be included recursively.
-setupRequired(meas_algorithms)
+setupRequired(meas_base)
```

Comment:
I agree. Similarly, please fix or remove this incorrect boilerplate. We no longer allow assuming recursive inclusion:
# Very common third-party packages (boost, python, swig, doxygen) 
# and very low-level LSST packages can be assumed to be recursively
# included by low-level LSST packages such as utils or daf_base.
# Any other package whose interface is used should be listed explicitly
# rather than assuming it will be included recursively.